### PR TITLE
Update dependencies to stable versions

### PR DESCRIPTION
**Pull request by NeonDaniel**

### CODE REVIEW

File: requirements/minimal.txt [hunk 1]
```diff
@@ -1,11 +1,12 @@
...
-ovos-config~=0.0,>=0.0.5a5
+ovos-utils~=0.0, >=0.0.26
+ovos-plugin-manager~=0.0, >=0.0.20a1
+# TODO: Stable OPM Release
+ovos-config~=0.0,>=0.0.5
 python-dateutil~=2.6
-ovos-lingua-franca~=0.4, >=0.4.6a1
-ovos_backend_client~=0.0, >=0.0.5a3
-ovos_workshop~=0.0, >=0.0.9a1
+ovos-lingua-franca==0.4.6a1
```
Comment (JarbasAl) [inline requirements/minimal.txt:9]: this should be >= probably, in general we always want to allow higher alpha versions or the stable releases as soon as it becomes available

Comment (NeonDaniel) [reply requirements/minimal.txt:9]: Alphas aren't necessarily stable, so I think it makes sense for a stable release to pin a known stable alpha; the next alpha ovos-core would be able to loosen this again

File: requirements/minimal.txt [hunk 2]
```diff
@@ -1,11 +1,12 @@
 requests~=2.26
 mycroft-messagebus-client~=0.9,!=0.9.2,!=0.9.3
 combo-lock~=0.2
-ovos-utils~=0.0, >=0.0.26a2
-ovos-plugin-manager~=0.0, >=0.0.19a4
-ovos-config~=0.0,>=0.0.5a5
+ovos-utils~=0.0, >=0.0.26
+ovos-plugin-manager~=0.0, >=0.0.20a1
+# TODO: Stable OPM Release
```
Comment (JarbasAl) [inline requirements/minimal.txt:6]: want to wait for this or merge as is? i think this can be bumped in #220 after rebasing

**Review by JarbasAl — APPROVED** [review]

File: requirements/minimal.txt (hunk 2 above)
Comment (NeonDaniel) [reply requirements/minimal.txt:6]: I guess as-is since this still needs validation

**Pull request merged**